### DISCUSSION TITLE
TShock 4.5.9 (Terraria 1.4.3.2)

### DISCUSSION THREAD

**Discussion by hakusaro** [Pryaxis/TShock#2511] [Show and tell]
Did you miss version 4.5.6? A lot of stuff changed in [TShock 4.5.6](https://github.com/Pryaxis/TShock/releases/tag/v4.5.6) so you might want to read those change notes too.

Did you miss version 4.5.7? Some stuff changed in [that release](https://github.com/Pryaxis/TShock/releases/tag/v4.5.7) too.

Did you miss version 4.5.8? Yep that also changed some stuff in [that release](https://github.com/Pryaxis/TShock/releases/tag/v4.5.8).

## Notable changes
* Added the ability to change a `TSPlayer`'s PVP mode. (@AgaSpace)
* Added protocol level support for Terraria 1.4.3.2. (@DeathCradle, @Patrikkk)

## Did you know?
[DeathCradle](https://github.com/sponsors/DeathCradle) now accepts GitHub sponsorships. Did you know that DeathCradle is the reason why TShock is about to switch to .NET 5 and/or 6 and ditch mono?

## Updating
To update TShock, simply replace all of the files in your server folder with those from the release folder.

<hr /><em>This discussion was created from the release <a href='https://github.com/Pryaxis/TShock/releases/tag/v4.5.9'>TShock 4.5.9 (Terraria 1.4.3.2)</a>.</em>

**Comment by HiddenStriker** [comment]
Thank you for the fast back to back releases. Terraria "journey's end" getting a huge update invalidating current version of tshock, then making 2 more updates constantly invalidating tshock was getting extremely annoying but you've managed to keep on top of it!

**Reply by hakusaro** [reply]
@HiddenStriker we've hoped and prayed for more updates since "Journey's End" (e.g., to fix [security issues we report](https://github.com/Pryaxis/TShock/security/advisories/GHSA-6w5v-hxr3-m2wx)) but it's always a surprise to us when the game gets updates too. I personally prefer a world where Re-Logic updates faster :)
